central/globaldb/telemetry.go

```diff
@@ -0,0 +1,42 @@
+package globaldb
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/postgres/pgadmin"
+	"github.com/stackrox/rox/pkg/postgres/pgconfig"
+	"github.com/stackrox/rox/pkg/sac"
+	"github.com/stackrox/rox/pkg/sac/resources"
+	"github.com/stackrox/rox/pkg/telemetry/phonehome"
+)
+
+// Gather database related information.
+// Current properties we gather:
+// "Database size"
+var Gather phonehome.GatherFunc = func(ctx context.Context) (map[string]any, error) {
+	ctx = sac.WithGlobalAccessScopeChecker(ctx,
```

Comment:
How is the access scope check enforced?

Reply:
Right, it's not needed. The functionality we call here doesn't even allow such file grained access level control.